invenio_app_ils/document_requests/views.py

```diff
@@ -88,70 +95,163 @@ def __init__(self, serializers, ctx, *args, **kwargs):
             setattr(self, key, value)
 
 
-class PendingRequestResource(DocumentRequestActionResource):
-    """Pending document request action resource."""
+class DocumentRequestDocumentResource(DocumentRequestActionResource):
+    """Endpoint to manage the Document of the Document Request."""
 
-    view_name = "{}_pending_request"
+    view_name = "{}_document"
 
     @pass_record
-    @need_permissions("document-request-pending")
+    @need_permissions("document-request-accept")
```

Comment:
Since all the actions have same permissions, maybe I would go for `document-request-actions` for all of them, WDYT?